src/dependencies/report.test.js

```diff
@@ -0,0 +1,85 @@
+describe('dependency reporting', () => {
```

Comment:
IMO the report UT should only test the report method, without calling the init and those tests should be part of wrapper.test.js

Reply:
Done in 733dd64